easydeasy_rest/api/serializers.py

```diff
@@ -33,7 +28,25 @@ class Meta:
 
 
 class ArticleSerializer(ModelSerializer):
+
+    def create(self, validated_data):
+        print(validated_data)
+
+        # Updates the to generic image if possible
+        if validated_data['img'] is None:
+            # Gets First Disease in list
+            disease: Disease = validated_data['diseases'][0]
+
+            image_folder = '/images/' + disease.category.name + '/'
```

Comment:
use f-string, in general in python 3.6+ f-string is always prefered, faster and more efficient and more understandable :D 

so - f'/images/{disease.category.name}/'